BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -87,6 +87,25 @@ public async void ApproveCharter_WithoutMembershipId_DisplaysErrorMessage()
         FlashMessages.Should().Contain("Membership ID is required.");
     }
 
+    [Fact]

```

Comment:
Could we turn this test into a `Theory` and have four different test cases (one for each of the possible results returned by the call to `LandlordService`)?
Should also check the result of the call to the controller is the correct thing, i.e. `RedirectToActionResult`

Reply:
I think I've figured it out but I'm not certain, I'll push what I have.